packages/node_modules/@ciscospark/widget-recents/src/components/RecentsHeader.js

```diff
@@ -4,54 +4,47 @@ import classNames from 'classnames';
 
 import {Button, Icon, SearchInput} from '@collab-ui/react';
 
-import PresenceAvatar from '@ciscospark/react-container-presence-avatar';
-
-
+import UserProfileAvatar from './UserProfileAvatar';
 import styles from './RecentsHeader.css';
 
 const propTypes = {
   currentUser: PropTypes.shape({
     id: PropTypes.string,
-    displayName: PropTypes.string
+    displayName: PropTypes.string,
+    email: PropTypes.string,
+    img: PropTypes.string
   }).isRequired,
   enableAddButton: PropTypes.bool.isRequired,
   enableSpaceListFilter: PropTypes.bool.isRequired,
   enableUserProfile: PropTypes.bool.isRequired,
+  enableUserProfileMenu: PropTypes.bool.isRequired,
   hideBottomBorder: PropTypes.bool.isRequired,
   onAddClick: PropTypes.func,
   onFilterChange: PropTypes.func,
-  onProfileClick: PropTypes.func
+  onProfileClick: PropTypes.func,
+  onSignOutClick: PropTypes.func
 };
 
 const defaultProps = {
   onAddClick: () => {},
   onProfileClick: () => {},
-  onFilterChange: () => {}
+  onFilterChange: () => {},
+  onSignOutClick: () => {}
 };
 
 function RecentsHeader(props) {
   const {
-    currentUser,
     enableAddButton,
     enableSpaceListFilter,
-    enableUserProfile,
     hideBottomBorder,
     onAddClick,
-    onFilterChange,
-    onProfileClick
+    onFilterChange
   } = props;
 
   return (
     <div className={classNames('ciscospark-recents-header', styles.recentsHeader, 'cui--dark', hideBottomBorder ? '' : styles.bottomBorder)}>
       <div className={classNames('ciscospark-recents-header-left', styles.headerSideItem)}>
-        {
-          enableUserProfile &&
-          <PresenceAvatar
-            avatarId={currentUser.id}
-            name={currentUser.displayName}
-            onClick={onProfileClick}
-          />
-        }
+        {<UserProfileAvatar {...props} /> }
```

Comment:
Can we specifically pass the props here? We typically only pass the props object when we have react components that are meant to be more of a higher-order component type wrapper component. This helps us know if we have prop naming issues in the future with changes.

Reply:
yeah I can do that.